examples/digital_fingerprinting/production/docker-compose.yml

```diff
@@ -109,7 +109,7 @@ services:
       target: runtime
       args:
         - MORPHEUS_CONTAINER=${MORPHEUS_CONTAINER:-nvcr.io/nvidia/morpheus/morpheus}
-        - MORPHEUS_CONTAINER_VERSION=${MORPHEUS_CONTAINER_VERSION:-v24.06.00-runtime}
+        - MORPHEUS_CONTAINER_VERSION=${MORPHEUS_CONTAINER_VERSION:-v24.06.00rc1-runtime}
```

Comment:
These shouldn't be needed, if you want to build the DFP container with a custom version use the `MORPHEUS_CONTAINER_VERSION` environment variable.

Reply:
Reverted to `24.06.00`